constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -109,6 +109,21 @@ public ConstructorIO(String apiToken, String apiKey, boolean isHTTPS, String hos
       this(apiToken, apiKey, isHTTPS, host, null);
     }
 
+    /**
+     * Sets apiKey
+     *
+     * @return true
+     * @throws ConstructorException
+     */
+    public boolean setApiKey(String apiKey) throws ConstructorException {
+        try {
```

Comment:
A try/catch here appears to be a little heavy.  I might suggest just `this.apiKey = apiKey;`